src/cascadia/TerminalApp/App.xaml

```diff
@@ -1,51 +1,59 @@
+-->
 <Toolkit:XamlApplication
     x:Class="TerminalApp.App"
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:TerminalApp"
-    xmlns:Toolkit="using:Microsoft.Toolkit.Win32.UI.XamlHost"
     xmlns:TA="using:TerminalApp"
+    xmlns:Toolkit="using:Microsoft.Toolkit.Win32.UI.XamlHost"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:TerminalApp"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d">
-    <!-- If you want to prove this works, then add `RequestedTheme="Light"` to
-    the properties on the XamlApplication -->
+    <!--
+        If you want to prove this works, then add `RequestedTheme="Light"` to
+        the properties on the XamlApplication
+    -->
     <Toolkit:XamlApplication.Resources>
 
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
-                <!-- Include the MUX Controls resources -->
-                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"/>
+                <!--  Include the MUX Controls resources  -->
+                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
                 <ResourceDictionary>
 
-                    <!-- We're going to apply this style to the root Grid acting
-                    as the tab row, because we need to be able to set its
-                    `Background` property to "{ThemeResource
-                    ApplicationPageBackgroundThemeBrush}" so it will be colored
-                    appropriately for the theme, regardless of what we set the
-                    RequestedTheme to -->
+                    <!--
+                        We're going to apply this style to the root Grid acting
+                        as the tab row, because we need to be able to set its
+                        `Background` property to "{ThemeResource
+                        ApplicationPageBackgroundThemeBrush}" so it will be colored
+                        appropriately for the theme, regardless of what we set the
+                        RequestedTheme to
+                    -->
                     <Style x:Name="BackgroundGridThemeStyle" TargetType="Grid">
                         <Setter Property="Background" Value="{ThemeResource ApplicationPageBackgroundThemeBrush}" />
                     </Style>
 
-                    <!-- We need to manually create the error text brush as a
-                    theme-dependent brush. SystemControlErrorTextForegroundBrush
-                    is unfortunately static. -->
+                    <!--
+                        We need to manually create the error text brush as a
+                        theme-dependent brush. SystemControlErrorTextForegroundBrush
+                        is unfortunately static.
+                    -->

```

Comment:
I don't totally understand why this file changed

Reply:
Yes. I don't either.